src/components/CardDropdownButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Button } from "components/Button";
+import { EllipsisBtnCopy } from "components/styles/Button";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+interface Props {
+  disabled?: boolean;
+  setIsVisibleCard?: (v: boolean) => void;
+  loading?: boolean;
+  isVisibleCard?: boolean;
+  cardItems: JSX.Element[];
+  dataCyBtn?: string;
+  dataCyCard?: string;
+}
+
+export const CardDropdownButton = ({
+  disabled = false,
+  loading = false,
+  setIsVisibleCard = function noop(v: boolean) {
+    return;
+  },
```

Comment:
does `() => {}` not work here as a noop?

Reply:
@typescript-eslint/no-empty-function prevents it